lib/safe-pg-migrations/plugins/statement_insurer/add_column.rb

```diff
@@ -59,9 +52,11 @@ def backfill_column_default(table_name, column_name)
         model = Class.new(ActiveRecord::Base) { self.table_name = table_name }
         quoted_column_name = quote_column_name(column_name)
 
+        Helpers::Logger.say_method_call(:backfill_column_default, table_name, column_name)
+
         Helpers::BatchOver.new(model).each_batch do |batch|
-          batch
-            .update_all("#{quoted_column_name} = DEFAULT")
+          without_statement_timeout { batch.update_all("#{quoted_column_name} = DEFAULT") }
```

Comment:
I think in the future that could be challenged: we should be able to set a timeout per batch as the number of row is under control.

Reply:
I can do that indeed.